modules/internal/ChapelArray.chpl

```diff
@@ -3242,11 +3242,23 @@ module ChapelArray {
 
     pragma "no doc"
     proc writeThis(f) throws {
+      var arrayStyle = f.styleElement(QIO_STYLE_ELEMENT_ARRAY);
+      var ischpl = arrayStyle == QIO_ARRAY_FORMAT_CHPL && !f.binary();
+      if rank > 1 && ischpl {
+        throw new owned IllegalArgumentError("Can not perform Chapel write of multidimensional array.");
```

Comment:
Modern English suggests to me that this should be "Cannot" rather than "Can not".  Was this wording intentional (i.e., is there a school of thought that prefers "can not"), or just something that slipped in?

Reply:
Just slipped in, fixed in: b46d1fcac2e3ef6c66b09661316165d57566d4c9